gcip/core/rule.py

```diff
@@ -13,6 +27,8 @@
 
 
 class WhenStatement(Enum):
+    """This enum holds diferent [when](https://docs.gitlab.com/ee/ci/yaml/#when) statements for `Rule`s."""
```

Comment:
```suggestion
    """Enum holds diferent [when](https://docs.gitlab.com/ee/ci/yaml/#when) statements for `Rule`s."""
```

Reply:
I really mean "this enum" _WhenStatement_ and not any enum.

But I found a typo

```suggestion
    """This enum holds different [when](https://docs.gitlab.com/ee/ci/yaml/#when) statements for `Rule`s."""
```

Reply:
Ok.